packages/phrases/src/locales/en.ts

```diff
@@ -20,9 +20,82 @@ const translation = {
     set_up: 'Set up',
     customize: 'Customize',
   },
+  main_flow: {
```

Comment:
main_flow phrases should not appear, a `rebase` may be required

Reply:
updated